README.md

```diff
@@ -27,33 +27,34 @@ Shell-operator provides:
 
 Steps to setup Shell-operator in your cluster are:
 - build an image with your hooks (scripts)
-- create necessary RBAC objects (for onKubernetesEvent binding)
-- run Pod with a built image
+- create necessary RBAC objects (for `kubernetes` bindings)
+- run Pod or Deployment with a built image
 
 ### Build an image with your hooks
 
 A hook is a script that, when executed with `--config` option, returns configuration in JSON. Learn [more](HOOKS.md) about hooks.
 
 Let's create a small operator that will watch for all Pods in all Namespaces and simply log a name of a new Pod.
 
-"onKubernetesEvent" binding is used to tell shell-operator about what objects we want to watch. Create the `pods-hook.sh` file with the following content:
+"kubernetes" binding is used to tell Shell-operator about what objects we want to watch. Create the `pods-hook.sh` file with the following content:
```

Comment:
>"kubernetes" binding is used to tell Shell-operator about what objects we want to watch. Create the `pods-hook.sh` file with the following content:

"kubernetes" binding is used to tell Shell-operator about objects that we want to watch. Create the `pods-hook.sh` file with the following content: